modules/release-notes/pages/relnotes.adoc

```diff
@@ -5,558 +5,33 @@
-
-| https://issues.couchbase.com/browse/MB-48211[MB-48211^]
-| *Summary*: XDCR - File descriptor leak in XDCR
-
-|===
-
-==== Tools, Web Console (UI), and REST API
-
-[#table_fixedissues_v702-tools-ui-rest-api,cols="25,66"]
-|===
-|Issue | Description
-
-| https://issues.couchbase.com/browse/MB-47001[MB-47001^]
-| *Summary*: Add Charts to Query Workbench
-
-| https://issues.couchbase.com/browse/MB-48081[MB-48081^]
-| *Summary*: cbbackupmgr start and end validations is a bit too aggressive
-
-|===
-
-==== Install and Upgrade
-
-[#table_fixedissues_v702-install-upgrade,cols="25,66"]
-|===
-|Issue | Description
-
-
-| https://issues.couchbase.com/browse/MB-47806[MB-47806^]
-| *Summary*: 7.0 Windows installer always rollbacks during install
-
-| https://issues.couchbase.com/browse/MB-48783[MB-48783^]
-| *Summary*: Offline upgrade from 7.0.0 or 7.0.1 on Debian and Ubuntu package install corrupts config files
-
-|===
-
-==== Storage
-
-[#table_fixedissues_v702-storage,cols="25,66"]
-|===
-|Issue | Description
-
-| https://issues.couchbase.com/browse/MB-46490[MB-46490^]
-| *Summary*: More memory overhead for non-collection index
-
-| https://issues.couchbase.com/browse/MB-47205[MB-47205^]
-| *Summary*: closeForRecovery does not release gCtx
-
-| https://issues.couchbase.com/browse/MB-47355[MB-47355^]
-| *Summary*: Compact parent page after page-split
-
-| https://issues.couchbase.com/browse/MB-47354[MB-47354^]
-| *Summary*: compact when marshalling full page over purge ratio
-
-| https://issues.couchbase.com/browse/MB-47429[MB-47429^]
-| *Summary*: Detect missing log file segment during initialization
-
-| https://issues.couchbase.com/browse/MB-47503[MB-47503^]
-| *Summary*: Recovered instances do not compact
-
-| https://issues.couchbase.com/browse/MB-47986[MB-47986^]
-| *Summary*: Log specific index stats at regular interval
-
-| https://issues.couchbase.com/browse/MB-47990[MB-47990^]
-| *Summary*: lss_rea_bytes and bytes_written do not always match perf html report
-
-| https://issues.couchbase.com/browse/MB-47992[MB-47992^]
-| *Summary*: Plasma Stats Compact Counter not updated on CompactFullMarshal
-
-| https://issues.couchbase.com/browse/MB-48356[MB-48356^]
-| *Summary*: MVCCPurger may stop running if doProceed check fails
-
-| https://issues.couchbase.com/browse/MB-47987[MB-47987^]
-| *Summary*: instMap in StatsLogger.run() should not be indexed by PlasmaId
-
-|===
-
-==== Views
+==== XDCR
 
-[#table_fixedissues_v702-views,cols="25,66"]
+[#table_fixedissues_v71-clustermanager,cols="25,66"]
 |===
 |Issue | Description
 
+| https://issues.couchbase.com/browse/MB-9982[MB-9982^]
+| *Summary*: XDCR should be incremental on topology changes.
 
-| https://issues.couchbase.com/browse/MB-47094[MB-47094^]
-| *Summary*: Slow processing of audit messages might lead to increase in RSS memory
+XDCR checkpointing is now entirely persistent through topology-changes on the target cluster.
```

Comment:
Change "target cluster" to "source cluster" on line 34. 

XDCR checkpointing is now entirely persistent through topology-changes on the source cluster.
This provides improved XDCR performance when failover and rebalance occur on the source cluster.

Reply:
Thanks, have fixed.